Makefile.in

```diff
@@ -4,19 +4,23 @@ AS		?= $(CC)
 AR		= ar
 RM		= rm -f
 
-CFLAGS	+= -Wall -Werror -Wextra -fpic -fno-builtin -fno-stack-protector -nostdlib \
-			 -nostdinc -Iinclude
+BASE_CFLAGS	+= -Wall -Werror -Wextra
+LIBC_CFLAGS = $(BASE_CFLAGS) -Iinclude -nostdlib -nostdinc -fno-stack-protector -fno-builtin -fpic
+SYSDEPS_CFLAGS += $(BASE_CFLAGS) -fno-stack-protector -fpic
+
 ifneq ($(CC), tcc)
-CFLAGS	+= -ansi -pedantic
+BASE_CFLAGS	+= -ansi -pedantic
 endif
 LDFLAGS	+= -nostdlib -shared
 ARFLAGS	= rcs
 
+
 SYSDEPS_PATH	= %(SYSDEPS_SRC)
```

Comment:
(also ping to @d0p1s4m4)
I think that this variable is not that useful :thinking:
Shall we remove it ?

Reply:
it's cleaner this way